### PR TITLE
Explain better why -showBuildSettings command takes a lot of time.

### PR DESCRIPTION
**Pull request by lpusok**
### Checklist
- [x] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [x] `step.yml` and `README.md` is updated with the changes (if needed)

### Version
Requires a *MINOR* [version update](https://semver.org/)

### Context

Improves the perceived slowness of the manage code signing phase; and adds log timestamps for debugging.

Resolves: https://bitrise.atlassian.net/browse/STEP-1834

### Changes

* Explicitely run -resolvePackageDependencies command to fetch Swift Packages.
* Adds timestamps and log -showBuildSettings commands.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -42,6 +44,22 @@ func main() {
...
 
+	xcodebuildVersion, err := utility.GetXcodeVersion()
+	if err != nil {
+		failf("Failed to determine Xcode version: %s", err)
+	}
+	logger.Printf("%s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
+
+	logger.Println()
+	if xcodebuildVersion.MajorVersion >= 11 {
+		// Resolve Swift package dependencies, so running -showBuildSettings later is faster later
```
Comment (Bence1001) [inline main.go:55]: `later` is duplicated.

File: main.go [hunk 2]
```diff
@@ -42,6 +44,22 @@ func main() {
 
 	cmdFactory := command.NewFactory(env.NewRepository())
 
+	xcodebuildVersion, err := utility.GetXcodeVersion()
+	if err != nil {
+		failf("Failed to determine Xcode version: %s", err)
+	}
+	logger.Printf("%s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
+
+	logger.Println()
```
Comment (shams-ahmed) [inline main.go:53]: is this for newline?

**Review by shams-ahmed — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (lpusok) [reply main.go:53]: Yes, for better log separation.

File: main.go (hunk 1 above)
Comment (lpusok) [reply main.go]: Fixed.

**Pull request merged**